opae-libs/.github/workflows/ccpp.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     - name: Get Packages
```

Comment:
This file lives in opae-libs, so it must be updated at https://github.com/OPAE/opae-libs.git.

Reply:
Done the changes and pushed to opae-libs.git